### PR TITLE
LPS-78399 Create a new Configuration: OrganizationLinkConfiguration

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-78399](https://issues.liferay.com/browse/LPS-78399).

### CODE REVIEW

File: modules/apps/foundation/organizations/organizations-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+organization-link-configuration-name=Organization Links
+organization-link-enabled-description=Check to enable defining arbitrary links between organizations.
+organization-link-types=Organization Link Types
+organization-link-types-description=The name of an Organization Link type
```
Comment (pei-jung) [inline modules/apps/foundation/organizations/organizations-api/src/main/resources/content/Language.properties]: I think the description message should be more descriptive, and it should be a complete sentence.

**Comment by pei-jung** [comment]
Hey Drew, the changes look good to me except for one language key. Can you fix that? Thanks!

**Comment by drewbrokke** [comment]
Sounds good.  I'll fix and send straight to Brian?

**Comment by pei-jung** [comment]
Yes, please send straight to Brian when you're done. Also note that we'll need to update the configuration category once Jorge's change for the System Setting get merged in (https://issues.liferay.com/browse/LPS-75377).

**Comment by drewbrokke** [comment]
Oh, right, thanks for reminding me!

**Pull request closed without merging**